examples/color/color_demo.py

```diff
@@ -3,7 +3,7 @@
 Color Demo
 ==========
 
-Matplotlib gives you 8 ways to specify colors,
+Matplotlib gives you 9 ways to specify colors,
 
 1) an RGB or RGBA tuple of float values in ``[0, 1]`` (e.g. ``(0.1, 0.2, 0.5)``
    or  ``(0.1, 0.2, 0.5, 0.3)``).  RGBA is short for Red, Green, Blue, Alpha;
```

Comment:
I’d say tab10: 

Are these all really different “ways” to specify colours?  If we are to have all these palettes I suggest specifying a palette is another way and then list all the palettes as a sub list.

Reply:
> and then list all the palettes as a sub list

Yes that is a very good idea.